css/style.css

```diff
@@ -6,3 +6,68 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+html,
+body {
+  background-image: url("https://img.freepik.com/free-photo/pastel-pack-cardboard-sheets_23-2148320477.jpg?w=826");
+}
+
+article:first-child {
+  color: blueviolet;
+}
+
+h1 {
+  text-align: center;
+  font-size: 40px;
+  font-family: sans-serif;
+}
+
+h2 {
+  text-align: center;
+  font-size: 25px;
+}
+
+h3,
+h4 {
+  text-align: center;
+  font-size: 25px;
+  font-family: sans-serif;
+}
+
+a {
+  display: flex;
+  justify-content:flex-start
+  font-size: 20px;
+  color: black;
+}
+
+.about {
+  font-style: italic;
+  text-align: center;
+}
+
+.about-me,
```

Comment:
It's fine to give them separate classes; but it's also fine to re-use the same class for all, maybe named for its appearance eg `.content-box {}` or `.bordered-box {}`